docs/using-gitbase/examples.md

```diff
@@ -140,13 +132,17 @@ Also, you can take a look to all the UDFs and their signatures in the [functions
 ## Retrieving UASTs with the UDF `uast`
 
 ```sql
-SELECT file_path, uast(blob_content, language(file_path)) FROM files;
+SELECT file_path,
+       uast(blob_content, language(file_path))
+FROM files;
 ```
 
 This function allows you to directly filter the retrieved UAST by performing a XPATH query on it:
 
 ```sql
-SELECT file_path, uast(blob_content, language(file_path), "//uast:FunctionGroup") FROM files;
+SELECT file_path,
```

Comment:
The same, maybe is worth it to filter by HEAD commits only and get the tokens using UAST_EXTRACT udf.

Reply:
I could not get anything with `uast` what makes sense and is usable ;-(
Some queries I've just wrapped by `uast_extract`. 
It even proofs me that everything around querying `uast` is messed up (all samples in engine etc.). Returning binary data doesn't make sense to me and it's not intuitive.

Reply:
PTAL on update.

Reply:
@kuba-- we were returning UAST data as json, but it was really slow compared with this new serialization.